src/list-data/request-action-mapping.ts

```diff
@@ -100,22 +100,22 @@ export const BusinessLookupRequestActions = [
 /** Entity types that require business lookup. */
 export const BusinessLookupEntityTypes = [
   EntityTypes.A,
-  EntityTypes.BC,
-  EntityTypes.BEN,
+  EntityTypes.BC, // Benefit Company
+  EntityTypes.BEN, // invalid?
   EntityTypes.C,
   EntityTypes.CC,
   EntityTypes.CCC,
   EntityTypes.CUL,
   EntityTypes.CP,
-  EntityTypes.CR,
+  EntityTypes.CR, // BC Limited Company
+  EntityTypes.FR, // Sole Proprietorship
```

Comment:
I tested FR by creating a name change NR with FM1056109 and verifying that corpnum was populated.